ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -32,9 +32,8 @@ enum TargetPlatform: CaseIterable {
     case .iOSDevice: return [.armv7, .arm64]
     // Include arm64 slices in the simulator for Apple silicon Macs.
     case .iOSSimulator: return [.i386, .x86_64, .arm64]
-    // TODO: Evaluate arm64 slice for Catalyst Apple silicon Macs, and x86_64h. Previous builds were
-    // limited to x86_64.
-    case .catalyst: return [.x86_64]
+    // TODO: Evaluate x86_64h slice. Previous builds were limited to x86_64.
```

Comment:
What's the difference between `x86_64h` and `x86_64` for Catalyst? Are they equivalent?

Does the Xcode 12 ignoring happen silently?

Reply:
I believe `x86_64h` is a sub-architecture of `x86_64` for Haswell+ Intel Macs. We originally identified it but the previous zip builds were still using `x86_64`.

Reply:
And yes, the Xcode 12 ignoring is silent. Only appears in the `xcodebuild` output log.